wiki/Tournaments/RMoT_Invitational/1/en.md

```diff
@@ -0,0 +1,115 @@
+# Rapid Monthly osu! Tournament 1
+
+The Rapid Monthly osu! Tournament Invitational \#1 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
```

Comment:
Nitpick. 4 -> four.

`It is an exclusive tournament where only players who reached semi-finals in one of the last four RMoT's get invited.`